src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -328,9 +328,13 @@ public String getEditedAtText(Boolean isGiverAnonymous) {
             String displayTimeAs = TimeHelper.formatDateTimeForComments(this.lastEditedAt);
             return "(last edited " +
                     (isGiverAnonymous ? "" : "by " + this.lastEditorEmail + " ") +
-                    "at " + displayTimeAs + ")";
+                    "at <span class=\"client-time\">" + displayTimeAs + "</span>)";
         } else {
             return "";
         }
     }
+
+    public String getCreatedAtText() {
```

Comment:
@damithc This method returns the created at time as text. The method naming follows the existing method written by someone else (maybe one of the developers) for getting the edited at time as text (<code>getEditedAtText</code>) and the attribute names are <code>createdAt</code> and <code>editedAt</code>.

If you want me to change this method name to <code>getCreatedTimeAsText</code>, I can do it with no problems, but I might want to change the <code>getEditedAt</code> method's name as well to <code>getEditedTimeAsText</code> to preserve uniformity.

Which one do you prefer?